bin/install-qa-check.d/60pkgconfig

```diff
@@ -13,6 +13,12 @@ pkgconfig_check() {
 		eqawarn "QA Notice: pkg-config files with wrong LDFLAGS detected:"
 		eqawarn "${f//${D}}"
 	fi
+
+	# Validate using pkgconfig
+	if ! pkg-config --validate "${files[@]}" ; then
```

Comment:
Is the `--validate` option portable across different pkg-config implementations? Hmm, I suppose we only support pkgconf these days, so feel free to ignore the problem. OTOH, you could check for pkgconf being available and just call it directly.

Reply:
OpenBSD has it and so does fd.o, so let's go with this for now, but I'll add a comment that it's possible some less-common impls (like some pure Perl ones, etc) may not implement it